### DISCUSSION TITLE
不是我说，这些牲口就活该烂尾

### DISCUSSION THREAD

**Discussion by ghost** [WeNeedHome/SummaryOfLoanSuspension#1048] [General]
实话实说，这些牲口烂尾是应该的，尤其是郑州的楼盘就tm的该烂尾，你不烂尾，天理不容。

**Comment by ghost** [comment]
mlgb

**Reply by GithubRobot01** [reply]
你就应该夭折,长这么大就会狺狺狂吠

**Reply by ghost** [reply]
hhh，你是烂尾了吗？在这叫，你的楼盘烂就是烂一辈子，等着政府把你的楼炸了吧，你还得还贷款，笑死我了。

**Reply by GithubRobot01** [reply]
你肯定是个孤儿吧,见不得别人好,内心扭曲

**Reply by JerryPlayer123** [reply]
滚你mlgb的张口就骂人咒人家死，牲口都没你这么恶毒

**Reply by GithubRobot01** [reply]
> 滚你mlgb的张口就骂人咒人家死，牲口都没你这么恶毒

你是哪来的狗东西?我骂的人是你爹?

**Comment by ytx0574** [comment]
你这说的, 事儿没在你身上,  买房的人掏干钱包活该吗? 该死的这是政府部门, 关键是你在天朝拿他一点办法没有

**Reply by ZerodLee** [reply]
你在别地方有办法吗，面对大资本照样被拿捏，连个特斯拉公司都一点办法都没有，世界上大部分人都是被管控的，没啥自由可言

**Comment by tanuk1997** [comment]
说得对，主人拿你点钱花花怎么了？让你活着就不错了，不知道感恩的东西

**Comment by Elijah-Wan** [comment]
继续烂下去，烂到所有改良派也对它失去幻想

**Reply by itsoursduty** [reply]
这就是所谓的加速主义了

**Reply by mand-byte** [reply]
> 继续烂下去，烂到所有改良派也对它失去幻想
没有改良派 只有当权派和一边派

**Comment by 1749579338** [comment]
买什么房，贷款推高房价害人害己
没钱就老实去做奴隶
肥了那些狗操的畜生
